symplyphysics/laws/optics/radius_newton_ring_from_wavelength_order_radius_curvature_refractive_index.py

```diff
@@ -0,0 +1,57 @@
+from sympy import (Eq, solve, sqrt)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless,
+)
+
+# Description
+## Newton's rings are annular interference maxima and minima that appear around the point of contact between
+## a convex lens and a plane—parallel plate when light passes through the lens and plate.
+## The order of interference indicates the number of whole wavelengths. When the difference in the path of the
+## beam reflected from the convex surface of the lens at the glass-air boundary and the beam reflected from the
+## plate at the air-glass boundary is equal to an entire wave "n", then these two waves hit the observation
+## point with the same faces and demonstrate interference.
+
+## Law is: r = sqrt(k * R * L / n), where
+## r - radius of Newton's ring,
```

Comment:
Dark or bright?

Reply:
> Dark or bright?

The formula itself is for dark rings, but if you substitute fractional values instead of integers instead of k, then you can get the radii of light rings. So if we're going to use this formula for both types of wheels, then we can't say for sure which ring it is.